src/xhr.js

```diff
@@ -17,6 +17,10 @@ import {
 
 const xhrFactory = function() {
   const xhr = function XhrFunction(options, callback) {
+    
+    // allow use cookie
+    options.withCredentials = true;
```

Comment:
This is available through options: https://github.com/videojs/videojs-contrib-hls#withcredentials